include/SFML/Audio/SoundRecorder.hpp

```diff
@@ -145,6 +145,34 @@ class SFML_AUDIO_API SoundRecorder : AlResource
     const std::string& getDevice() const;
 
     ////////////////////////////////////////////////////////////
+    /// \brief Set the channel count of the audio capture device
+    ///
+    /// This method allows you to set the channel count of the
+    /// recording. Currently only 16-bit mono and 16-bit stereo
+    /// are supported.
+    ///
+    /// \param channelCount Integer specifying the channel count.
+    ///                     Currently only mono and stereo are
+    ///                     supported.
+    ///
+    /// \see getChannelCount
+    ///
+    ////////////////////////////////////////////////////////////
+    void setChannelCount(unsigned int channelCount);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the number of channels used by this recorder
+    ///
+    /// Returns the number of channels used by this recorder.
```

Comment:
DRY again: The info that something is returned is already in the brief description. Also it should read "[...]mono and stereo **are** supported[...]"